Update README.md (#29)
nuxt-community/separate-env-module: README.md

## Caveats

> **IMPORTANT:** Be aware that `server` is included on the first render of your application.

Reviewer: ```suggestion
**IMPORTANT:** Be aware that `server-side` means on **every** first render of your application.
```

That's what I actually meant :P

> Your secret tokens won't be included anywhere **except where you use them **

Reviewer: ```suggestion
Your secret tokens won't be included anywhere **except where you use them**
```

Author: Ah, okay. I misunderstood how `server-side` was used, since it looked like it was referring to a variable/value.
